install-repositories.sh

```diff
@@ -8,3 +8,11 @@ LOG_END='\n\e[0m'      # new line + reset color
 printf "${LOG_START}Initializing submodules...${LOG_END}"
 
 git submodule update --init --recursive --remote --rebase --force
+
+# We need to checkout `coverage-pools` at `6a9b084`, as after that commit we
+# introduced in the the `Asset Pool` contract a call to a `delegate` function
```

Comment:
How about:
We need to stay at commit `6a9b084` which is right before the `AssetPool` contract took a new collateral token with a `delegate()` function. Coverage pool utilizes Keep Token which does not have a `delegate()` function. If the latest code is used for coverage pools, then the CI will break during the deployment of the `AssetPool` contract.

Reply:
Yep, sounds better :) Changed in 8b88d506f3a577fe14d602504379a2b843f01661